src/Spectre/Models/UserModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
Those files (from *Spectre* project) were commited earlier, as a part of another task. Nevertheless, I will add those headers.